notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<p>peak of popularity of caffe was in 2018, for now it os almost dead framework. Maybe we can consider different model example?</p><p>e.g. some  model from here https://www.tensorflow.org/api_docs/python/tf/keras/applications</p><p>Additionally it contains preprocessing function, which we can use as reference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
here example how it can be used with openvino:
https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/101-tensorflow-to-openvino/101-tensorflow-to-openvino.ipynb

Reply:
changed to InceptionResNetV2